### PR TITLE
Improved with-ant-design example

### PR DESCRIPTION
**Pull request by notatestuser**
With minimal CSS (no more CDN) using `@zeit/next-css`

### CODE REVIEW

File: examples/with-ant-design/next.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const withCss = require('@zeit/next-css')
+
+// fix: prevents error when .css files are required by node
+if (typeof require !== 'undefined') {
+  require.extensions['.css'] = (file) => {}
```
Comment (timneutkens) [inline examples/with-ant-design/next.config.js]: This seems incorrect 🤔next-css should also handle the discard of styles on the server.

Comment (notatestuser) [reply examples/with-ant-design/next.config.js]: Agreed. But it breaks without this during SSR.

**Comment by ateufel** [comment]
wouldn´t it be better to add the css files to pages/_document.js, to make it available on every page? at least that´s how i am using it.

File: examples/with-ant-design/next.config.js (hunk 1 above)
Comment (timneutkens) [reply examples/with-ant-design/next.config.js]: Then there must be something wrong with the setup 🤔

Comment (timneutkens) [reply examples/with-ant-design/next.config.js]: Or it's related to antd not importing with `.css`

Comment (notatestuser) [reply examples/with-ant-design/next.config.js]: Here is what I get:
```
[...]/node_modules/antd/lib/style/index.css:6
@font-face {
^

SyntaxError: Invalid or unexpected token
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:152:10)
    at Module._compile (module.js:624:28)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/luke/binance/react-exchange/node_modules/antd/lib/radio/style/css.js:3:1)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
```

**Comment by notatestuser** [comment]
@ateufel not sure if that would mean that antd is included on error pages, etc.

**Comment by ateufel** [comment]
good point...although, it´s just the css file, and i would prefer to have "good looking" error pages :)

File: examples/with-ant-design/next.config.js (hunk 1 above)
Comment (notatestuser) [reply examples/with-ant-design/next.config.js]: @timneutkens they use their custom babel plugin to do it, so no, it's not a direct css import.

**Review by timneutkens — APPROVED** [review]

**Pull request merged**